README.md

```diff
@@ -1,3 +1,17 @@
+# Repository moved to https://repo.getmonero.org/monero-project/monero-site
+
+This repository has recently been moved to https://repo.getmonero.org/monero-project/monero-site  
+Please do your changes on the new repository.
+
+To contribute on https://repo.getmonero.org/ please proceed this way:
+
+* Login using OAuth2 with your GitHub account
```

Comment:
Don't think that using a GitHub account is necessary, people can just subscribe to GitLab

Reply:
I'll write both options then.